pkg/bmc/idrac_virtualmedia.go

```diff
@@ -86,3 +88,53 @@ func (a *redfishiDracVirtualMediaAccessDetails) RAIDInterface() string {
 func (a *redfishiDracVirtualMediaAccessDetails) VendorInterface() string {
 	return "no-vendor"
 }
+
+func (a *redfishiDracVirtualMediaAccessDetails) ValidateBootInterface(bmcCreds Credentials) (bool, error) {
+	var hasVirtualMedia, err = a.validateVirtualMedia(bmcCreds)
+	return hasVirtualMedia, err
+}
+
+func (a *redfishiDracVirtualMediaAccessDetails) validateVirtualMedia(bmcCreds Credentials) (bool, error) {
```

Comment:
Does this method need to be separate from ValidateBootInterface?

Reply:
Yeah, maybe I was over thinking.